pyproject.toml

```
@@ -12,7 +12,7 @@ homepage = "https://github.com/tr11/wagtail-graphql"
 keywords = ['wagtail', 'graphql', 'api', 'wagtail-graphql']
 
 [tool.poetry.dependencies]
-python = "~3.6||~3.7"
+python = "~3.6||~3.7||~3.8"
```

Could this be switched to 

```suggestion
python = "^3.6"
```

instead, to allow all minor Python versions above 3.6? I don’t really see the point in not allowing that, considering Python has quite a good versioning policy and track record of being backwards-compatible.

Practically, Python 3.9 is due to be released in October this year, so also feels weird to have to come back to this again in 3 months’ time.